app/scripts/vendor/crosstab.js

```diff
@@ -72,6 +76,11 @@
         if (!setItemAllowed) {
             reasons.push('localStorage.setItem not allowed');
         }
+        // begin FxA addition
+        if (hasBroadcastChannel) {
```

Comment:
I realise this change only follows the precedent already laid out, but I'm wondering whether this is the correct level of abstraction to test this. Might it be preferable to do these checks in the inter-tab channel so that this module can be left in harmony with the original 3rd-party copy? Or am I missing something which precludes doing that?